### ISSUE TITLE
list items provided under expanded button in the group are not reachable with keyboard navigation

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4629]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 2.1.1 Keyboard

**Test Environment:**
- Test URL: https://clarity.design/documentation/button-group
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 20.1.1

**Steps to reproduce:**
1. Open the test url.
2. Move focus onto one of the collapsed button (horizontal 3 dots button) in a group, expand that button by pressing enter key and try to navigate through the expanded list using up/down arrow keys.

**Expected result:**
When the button is expanded focus should be placed on the first item in the list which should be announced and user should be able to navigate through the list.

**Actual result:**
Button is expanded but focus is not placed on the first item in the list. When user tries to navigate through the list using arrow keys, the list closed and focus is placed on the next fosusable element.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
